modules/sttp/src/main/scala/pureconfig/module/sttp/package.scala

```diff
@@ -0,0 +1,18 @@
+package pureconfig.module
+
+import com.softwaremill.sttp._
+import pureconfig.ConfigReader
+import pureconfig.error.CannotConvert
+
+import scala.util.Try
+
+package object sttp {
+
+  implicit val reader: ConfigReader[Uri] =
+    ConfigReader.fromNonEmptyString { str ⇒
```

Comment:
Nitpick: can you use `=>` instead of `⇒`?

Reply:
Sure